test/crs/conversions.jl

```diff
@@ -506,6 +506,20 @@
       @inferred convert(LatLon, c2)
     end
 
+    @testset "Cartesian <> Mercator" begin
```

Comment:
Please, move this testset to before the `"Projection conversion"` testset, rename it to `"Cartesian <> Projected"` and add testes with one more `Projected` CRS.